Modifies the set of accepted characters in folder names for the cyrus imapd server [Version: 2.3.9]

diff -r 6bcd324910d3 imap/mboxname.c
--- a/imap/mboxname.c	Thu Oct 25 08:30:33 2007 +0200
+++ b/imap/mboxname.c	Thu Oct 25 08:30:43 2007 +0200
@@ -700,8 +700,13 @@ int mboxname_netnewscheck(char *name)
 /*
  * Apply site policy restrictions on mailbox names.
  * Restrictions are hardwired for now.
- */
+
+ * original definition 
 #define GOODCHARS " +,-.0123456789:=@ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz~"
+ */
+
+#define GOODCHARS " #$%'()*+,-.0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`abcdefghijklmnopqrstuvwxyz{|}~"
+
 int mboxname_policycheck(char *name)
 {
     unsigned i;
